subsys/net/lib/app/Makefile

```diff
@@ -4,9 +4,6 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-ccflags-$(CONFIG_NET_L2_BLUETOOTH) += -I${ZEPHYR_BASE}/samples/bluetooth/
```

Comment:
While reworking the lines in this file, I seemed to have removed the BT line too. That BT line is extra and should not be here anyway but clearly does not belong to this patch, I will send a new version.